lib/express-handlebars.js

```diff
@@ -190,11 +188,19 @@ ExpressHandlebars.prototype.render = function (filePath, context, options) {
 };
 
 ExpressHandlebars.prototype.renderView = function (viewPath, options, callback) {
-    var context = options,
-        data    = options.data;
+    options || (options = {});
+
+    var context = options;
+    var data    = options.data;
 
-    var helpers = utils.extend({},
-            this.handlebars.helpers, this.helpers, options.helpers);
+    var helpers = utils.extend({}, this.helpers, options.helpers);
+
+    var partials = Promise.all([
+        this.getPartials({cache: options.cache}),
+        Promise.resolve(options.partials)
+    ]).then(function (partials) {
+        return utils.extend.apply(null, [{}].concat(partials));
```

Comment:
Used `fn.apply()` here in case future code adds more sources the merging code won't have to change.